src/routes/docs/advanced/platform/billing/+page.markdoc

```diff
@@ -44,4 +44,14 @@ By default, an email alert will be sent at 75% of the budget cap. You can add up
 
 # Redeem credit {% #redeem-credit %}
 If you received a redeemable code for Appwrite Cloud credits, you can redeem them in the **Available credit** section.
-When you redeem credit, the credit balance will be automatically applied to the next billing cycle.
\ No newline at end of file
+When you redeem credit, the credit balance will be automatically applied to the next billing cycle.
+
+# Recurring payments for Indian developers {% #recurring-payments-for-indian-developers %}
+
+The Reserve Bank of India (RBI) mandates additional security measures for recurring payments on Indian cards. 
+Appwrite is obligated to ask for verification before billing your card. 
+Appwrite asks for verification for up to $150 in case you use add-ons, but will not charge more than the actual amount used.
```

Comment:
More information is need on budget cap, and another paragraph to contact us in the case where a higher limit is required.